packages/elements-core/src/components/Docs/HttpService/HttpService.tsx

```diff
@@ -64,7 +63,9 @@ const HttpServiceComponent = React.memo<HttpServiceProps>(({ className, data, he
       ) : (
         <Box mb={10}>
           {description}
-          <PoweredByLink source={data.name ?? 'no-title'} pathname={pathname} packageType="elements" headless />
+          {pathname && (
+            <PoweredByLink source={data.name ?? 'no-title'} pathname={pathname} packageType="elements" headless />
+          )}
```

Comment:
I also find it weird that this depends on the pathname being present :\

Reply:
If `pathname` is only a "nice to have" then I can make it optional in PoweredByLink. :shrug: 

But in practice it will work the same either way, so I chose simpler implementation.